README.md

```diff
@@ -321,51 +322,51 @@ The `/api/v1/export` endpoint should return the following response:
 
 We recommend using either [binary releases](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) or
 [docker images](https://hub.docker.com/r/victoriametrics/victoria-metrics/) instead of building VictoriaMetrics
-from sources. Building from sources is reasonable when developing an additional features specific
+from sources. Building from sources is reasonable when developing additional features specific
 to your needs.
 
 
 #### Development build
 
 1. [Install Go](https://golang.org/doc/install). The minimum supported version is Go 1.12.
 2. Run `make victoria-metrics` from the root folder of the repository.
-   It will build `victoria-metrics` binary and put it into the `bin` folder.
+   It builds `victoria-metrics` binary and put it into the `bin` folder.
 
 #### Production build
 
 1. [Install docker](https://docs.docker.com/install/).
 2. Run `make victoria-metrics-prod` from the root folder of the repository.
-   It will build `victoria-metrics-prod` binary and put it into the `bin` folder.
+   It builds `victoria-metrics-prod` binary and put it into the `bin` folder.
 
 #### ARM build
 
-ARM builds may run on Raspberry Pi or on [energy-efficient ARM servers](https://blog.cloudflare.com/arm-takes-wing/).
+ARM build may run on Raspberry Pi or on [energy-efficient ARM servers](https://blog.cloudflare.com/arm-takes-wing/).
```

Comment:
It was plural because there could be multiple ARM builds

Reply:
what this sentence for? Define the count or the ability

Reply:
Let's leave `build` for the consistency sake with the rest of the text.